### PR TITLE
Add deprecations and FileSeedProvider

### PR DESCRIPTION
**Pull request by jackwaudby**
* S3SeedProvider is deprecated as of 5.26
* FileSeedProvider added in 5.26
* file: is deprecated for URLConnectionProvider as of 5.26
* Reorder providers putting S3SeedProvider last

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:478]:
```suggestion
** `file:` label:deprecated[Deprecated in 5.26]
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:536]:
```suggestion
** `s3:` label:deprecated[Deprecated in 5.26] 
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:578]:
```suggestion
| `URLConnectionSeedProvider` label:deprecated[Deprecated in 5.26], +
`FileSeedProvider` label:new[Introduced in 5.26] 
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:594]:
```suggestion
| `S3SeedProvider` label:deprecated[Deprecated in 5.26], +
`CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:598]:
```suggestion
| `CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:602]:
```suggestion
| `CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:485]:
```suggestion
label:new[Introduced in 5.25], the `CloudSeedProvider` supports:
```

File: modules/ROOT/pages/clustering/databases.adoc
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:472]:
```suggestion
label:new[Introduced in 5.26], the `FileSeedProvider`  supports:
```

**Review by NataliaIvakina — APPROVED** [review]
@jackwaudby, thank you! This looks good. I added a few suggestions to be in line with the rest of the manual.
Please, do not merge this PR! We have to publish the 5.25 docs first.

**Comment by jackwaudby** [comment]
@NataliaIvakina thank you, I've committed the suggestions. Ok sure thing, I will leave it up to you to hit merge when the time comes

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Pull request merged**
